pkg/scheduler/testing/wrappers.go

```diff
@@ -1230,6 +1231,11 @@ func (wrapper *ResourceSliceWrapper) Device(name string, otherFields ...any) *Re
 	return wrapper
 }
 
+func (wrapper *ResourceSliceWrapper) ResourceSliceCount(count int) *ResourceSliceWrapper {
```

Comment:
I don't see this called anywhere... is it?

Reply:
Perhaps not now, but it seemed more consistent to have it.